lib/BabelPluginExportToFunction.js

```diff
@@ -18,15 +18,28 @@ const isDefaultExport = R.allPass([
     R.pathEq([ 'object', 'name' ], 'exports'),
     R.pathEq([ 'property', 'name' ], 'default')
 ]);
+const hasMoreThanTwoParameters = R.pathSatisfies(R.gte(R.__, 2), [ 'params', 'length' ]);
 const isValidFunctionExpression = R.allPass([
     R.propEq('type', 'FunctionExpression'),
-    R.pipe(R.prop('params'), R.length, R.gte(R.__, 2))
+    hasMoreThanTwoParameters
+]);
+const isValidFunctionDeclaration = R.allPass([
+    R.propEq('type', 'FunctionDeclaration'),
```

Comment:
This includes generator functions. Not sure if this could become a problem though.